### DISCUSSION TITLE
2D Kde not show a contour far from the 3sigma one

### DISCUSSION THREAD

**Discussion by jecampagne** [arviz-devs/arviz#2380] [Q&A]
Hello
Looking at [KDE 2D example demo]( https://python.arviz.org/en/stable/examples/plot_kde_2d_hdi.html)
![image](https://github.com/user-attachments/assets/c714d7a3-a0aa-4adf-a8d4-384a74385590)
I wander how to not display the very non-smooth contour located far from the 3-sigma contour.

Thanks

**Comment by aloctavodia** [comment]
I am almost sure we did not show that contour. So probably something changed in matplotlib... Anyway, in the meantime a quick trick is to make the contour transparent with  `contour_kwargs={"alpha":0}`

**Reply by OriolAbril** [reply]
The fact that it doesn't happen in <=0.17.1 seems to point to that: https://python.arviz.org/en/v0.17.1/examples/plot_kde_2d_hdi.html. We'll have to look at what has changed, especially when providing hdi_probs, only the requested contour lines should be plotted
